### PR TITLE
i18n(pt-br): Update `ui` labels

### PR DESCRIPTION
**Pull request by Fryuni**
#### Description

Update the Portuguese translations of the UI labels.

### CODE REVIEW

File: src/i18n/pt-br/ui.ts [hunk 1]
```diff
@@ -103,6 +104,9 @@ export default UIDictionary({
 	'feedback.categoryGroupLabel': 'Escolha a categoria do feedback',
 	'feedback.issue': 'Problema',
 	'feedback.createIssue': 'Criar um problema no GitHub',
+	'feedback.createIssue.description': 'A maneira mais rápida de alertar nosso time sobre um problema.',
+	'feedback.sendFeedback': 'Enviar feedback',
+	'feedback.sendFeedback.description': 'Nos envia um feedback diretamente.',
```
Comment (vitorxfs) [inline src/i18n/pt-br/ui.ts:109]:
```suggestion
	'feedback.sendFeedback.description': 'Nos envie um feedback diretamente.',
```

I think `envie` would make more sense in here.

Comment (Fryuni) [reply src/i18n/pt-br/ui.ts:109]: I think this is refurring to what the button does, not the user.

At least that is what I got from the other descriptions.

Comment (vitorxfs) [reply src/i18n/pt-br/ui.ts:109]: That makes sense. LGTM, then.

**Review by vitorxfs — APPROVED** [review]

File: src/i18n/pt-br/ui.ts [hunk 2]
```diff
@@ -125,4 +129,6 @@ export default UIDictionary({
 		'Quer contribuir para este guia? Tem uma postagem de blog, vídeo ou outro recurso para compartilhar sobre a migração desta tecnologia para o Astro?',
 	'stub.description.cms': 'Sabe mais sobre como usar este CMS com Astro?',
 	'stub.description.backend': 'Sabe mais sobre como usar este serviço de back-end com o Astro?',
+	'starlight.title': 'Quer criar sua própria Docs?',
```
Comment (Yan-Thomas) [inline src/i18n/pt-br/ui.ts:132]:
```suggestion
	'starlight.title': 'Quer criar sua própria documentação?',
```

File: src/i18n/pt-br/ui.ts (hunk 1 above)
Comment (Yan-Thomas) [inline src/i18n/pt-br/ui.ts:109]: Aqui o original é `Send us a message directly`, mudando um pouco fica mais próximo do original.
```suggestion
	'feedback.sendFeedback.description': 'Nos envie uma mensagem diretamente.',
```

File: src/i18n/pt-br/ui.ts [hunk 3]
```diff
@@ -103,6 +104,9 @@ export default UIDictionary({
 	'feedback.categoryGroupLabel': 'Escolha a categoria do feedback',
 	'feedback.issue': 'Problema',
 	'feedback.createIssue': 'Criar um problema no GitHub',
```
Comment (Yan-Thomas) [inline src/i18n/pt-br/ui.ts:106]: Não faz parte desse PR mas aqui deveria ser `Issue` ao invés de `problema`
```suggestion
	'feedback.createIssue': 'Criar uma Issue no GitHub',
```

**Review by Yan-Thomas — COMMENTED** [review]
Algumas pequenas sugestões.

File: src/i18n/pt-br/ui.ts (hunk 1 above)
Comment (Fryuni) [reply src/i18n/pt-br/ui.ts:109]: Going by the translations around this I was thinking that this was for a a description of a button. Now I saw it on the UI, so it makes sense.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**